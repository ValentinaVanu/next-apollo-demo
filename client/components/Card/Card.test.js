import { CardCollection } from '.'
import { Provider } from 'react-redux'
import renderer from 'react-test-renderer'

describe('Renders card collection component', () => {
  const store = {
    getState: () => ({
      sapient : {
        people: [],
        limit: 20,
      },
    }),
    subscribe: jest.fn(),
    dispatch: jest.fn(),
  }
  it("should render the component", () => {
    const { toJSON } = renderer.create(<Provider store={store}><CardCollection /></Provider>)

    expect(toJSON()).toMatchSnapshot()
  })
})
