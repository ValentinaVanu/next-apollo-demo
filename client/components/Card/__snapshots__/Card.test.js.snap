// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders card collection component should render the component 1`] = `
<div
  className="Cardstyle__StyledCardWrapper-sc-1aqocvc-1 kJfoTV"
/>
`;
